### PR TITLE
code review changes

### PR DESCRIPTION
**Pull request by navyapokuri**
Included name attributes in the input fields as this is important for server-side processing of the form data.
Updated JSON property name from "dst" to "district"
Updated public property name from "dst" to "District" and type from "dstUnion" to "DistrictUnion" to make them more understandable.
Modified alt attributes to more accurate descriptive for images.
Adding the justify-content-center class to center the images on the page.

**List three items you learned from the code review.**
Consistent Naming Conventions: The code review emphasizes the importance of maintaining consistent naming conventions for classes, IDs, and other HTML attributes.

Accessibility and Styling Enhancements:
The modification of alt attributes for images to make them more accurately descriptive indicates a focus on accessibility.

JSON Property and Public Property Improvements:
 updates to the JSON property name from "dst" to "district" and the public property name from "dst" to "District." These changes indicate a commitment to improving code readability and maintainability.

Was the project available in GitHub on time? Yes
Does the project compile? Yes
Does the project have adequate documentation? Yes

**three changes committed by me to our group's repository**
https://github.com/harry0878/IS7024/commit/107bb6d2bde479d53dcef4848fcc8d383908da6e

https://github.com/harry0878/IS7024/commit/31f3969e30d7866d32d11152211883151938606b

https://github.com/harry0878/IS7024/commit/7905cf2786fb693401103477598b5e57e3539a43

### CODE REVIEW

File: Incidents.cs [hunk 1]
```diff
@@ -37,8 +37,8 @@ public partial class Incidents
         [JsonConverter(typeof(ParseStringConverter))]
         public long? Ucr { get; set; }
 
-        [JsonProperty("dst")]
-        public DstUnion Dst { get; set; }
+        [JsonProperty("district")]
+        public DistrictUnion District { get; set; }
```
Comment (navyapokuri) [inline Incidents.cs:41]: Updated JSON property name from "dst" to "district"
Updated public property name from "dst" to "District" and type from "dstUnion" to "DistrictUnion" to improve code readability and maintainability.
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#linq-queries

File: Pages/Feedback.cshtml [hunk 2]
```diff
@@ -35,16 +35,16 @@
                         <div class="contact_form-container">
                             <div>
                                 <div>
-                                    <input type="text" placeholder="Full Name" />
+                                    <input type="text" placeholder="Full Name" name="fullName"/>
```
Comment (navyapokuri) [inline Pages/Feedback.cshtml:38]: Included name attributes in the input fields as this is important for server-side processing of the form data.
https://www.w3schools.com/html/html_form_input_types.asp

File: Pages/Index.cshtml [hunk 3]
```diff
@@ -74,30 +75,30 @@
 }
 
 <h3 class="text-center w-100 text-dark h3 mt-5 mb-2">Safest Neighborhoods In Cincinnati</h3>
-<div class="row g-3">
+<div class="row justify-content-center g-3">
```
Comment (navyapokuri) [inline Pages/Index.cshtml:78]: Adding the justify-content-center class to center the images on the page.

File: Pages/Index.cshtml [hunk 4]
```diff
@@ -74,30 +75,30 @@
...
     <div class="col-4 ">
-        <img src="https://i0.wp.com/haushomemagazine.com/wp-content/uploads/2021/04/My-Post-6.jpg?w=1200&ssl=1" class="zoom-hover rounded-top" alt="image of place" />
+        <img src="https://i0.wp.com/haushomemagazine.com/wp-content/uploads/2021/04/My-Post-6.jpg?w=1200&ssl=1" class="zoom-hover rounded-top" alt=">Hyde Park, Cincinnati" />
```
Comment (navyapokuri) [inline Pages/Index.cshtml:80]: Modified alt attributes value to more accurate descriptive for images.

File: Incidents.cs (hunk 1 above)
Comment (discospiff) [reply Incidents.cs:41]: Yes, very good idea.  Variable names should be self-describing.
Avoid the temptation to use abbreviations.  You'll forget what abbreviation you used!

File: Pages/Feedback.cshtml (hunk 2 above)
Comment (discospiff) [reply Pages/Feedback.cshtml:38]: Good.

File: Pages/Index.cshtml (hunk 4 above)
Comment (discospiff) [reply Pages/Index.cshtml:80]: Very good.  Consider describing Hyde Park, briefly.  What would a user see in this image?

**Comment by discospiff** [comment]
Very good suggestions.
I recommend merging this branch.

**Pull request merged**